cluster/src/test/java/io/scalecube/cluster/metadata/SimpleMapMetadataCodec.java

```diff
@@ -0,0 +1,35 @@
+package io.scalecube.cluster.metadata;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.nio.ByteBuffer;
+import java.util.Map;
+import reactor.core.Exceptions;
+
+public class SimpleMapMetadataCodec implements MetadataCodec<Map<String, String>> {
```

Comment:
should be  MetadataCodec<Map<String, ByteBuffer>> and not sting IMHO

Reply:
no, it's a client type